lib/Worker.js

```diff
@@ -134,6 +129,11 @@ module.exports = (function(){
                 // entry won't be created for it until the previous one's creation has completed
                 that.handledEntries[e.entryId] = 'pending';
                 logger.info("[%s] Entry started playing - creating a new object for it", e.entryId);
+                logger.debug("[%s] Removing from endlistAppender", e.entryId);
+                that.endListAppender.removeEntry(e.entryId)
+                    .catch(function() {
+                        logger.error("Error removing entry to endlistAppender");
+                    });
```

Comment:
Note that if createLiveEntry is failed, then entry can still be "offline" but without end-list